HabboHotel/Users/Profile/ProfileManager.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Dapper;
+using Plus.Database;
+using Plus.HabboHotel.Groups;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Plus.HabboHotel.Users.Profile
+{
+    public class ProfileManager : IProfileManager
+    {
+        private readonly IGroupManager _groupManager;
+        private readonly IDatabase _database;
+
+        public ProfileManager(IGroupManager groupManager, IDatabase database)
+        {
+            _groupManager = groupManager;
+            _database = database;
+        }
+
+        public List<Group> GetGroups(Habbo habbo)
+        {
+            var groups = _groupManager.GetGroupsForUser(habbo.Id);
+            return groups;
+        }
+
+        public Habbo? GetProfile(Habbo habbo)
+        {
+            var targetData = PlusEnvironment.GetHabboById(habbo.Id);
+            if (targetData == null)
+            {
+                habbo.GetClient().SendNotification("An error occured whilst finding that user's profile.");
+                return null;
+            }
+            var groups = _groupManager.GetGroupsForUser(targetData.Id);
```

Comment:
It gets groups but doesnt do anything.
And it returns the passed arguement Habbo again.
Makes no sense.

Reply:
Yes you are totally right. Reading through my code sometimes things don't make too much sense.